hack/tools/go.mod

```diff
@@ -2,4 +2,18 @@ module sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm/hack/tools
 
 go 1.12
 
-require sigs.k8s.io/controller-tools v0.2.0-rc.0
+require (
+	github.com/golangci/golangci-lint v1.17.1
+	sigs.k8s.io/controller-tools v0.2.0-rc.0
+)
+
+replace (
```

Comment:
Why do you need all these replace directives? I saw your question about GOPROXY and direct... I checked out your branch locally and did not receive the same errors you pasted in your comment. Can you take these out and see if it passes CI?

Reply:
I've tried before, but it failed!
I'll try again.